scripts/safe_template_linter.py

```diff
@@ -197,7 +197,7 @@ def print_results(self):
         """
         Prints the results represented by this rule violation.
         """
-        print "{}: {}".format(self.full_path, self.rule.rule_id)
+        print "{}: {}\n\n".format(self.full_path, self.rule.rule_id)
```

Comment:
I think the single line format is simpler for working with grep, but we can discuss.